scripts/build/utils.js

```diff
@@ -49,7 +49,7 @@ async function exec(cmd, args, options = {}) {
                     resolve({ exitCode });
                 }
                 else {
-                    reject(new Error(`Process exited with code: ${exitCode}`));
+                    reject(new Error(`${JSON.stringify({ cmd, args, options })}\nProcess exited with code: ${exitCode}`));

```

Comment:
TODO: this was me debugging a failed command; this should really print out the cmd+args because the error stack here doesn't work properly due to callbacks.